arcade/application.py

```diff
@@ -957,7 +957,7 @@ def has_sections(self) -> bool:
         else:
             return self.section_manager.has_sections
 
-    def add_section(self, section, at_index: Optional[int] = None, at_draw_order: Optional[int] = None) -> None:
+    def add_section(self, section: Optional[SectionManager], at_index: Optional[int] = None, at_draw_order: Optional[int] = None) -> None:
```

Comment:
```suggestion
    def add_section(self, section: Optional[SectionManager] = None, at_index: Optional[int] = None, at_draw_order: Optional[int] = None) -> None:
```

Reply:
@cspotcode It might break some code. I don't know if we should do that.

Reply:
Oh right, it was required before, should be required after.